src/components/FeatureViews/Registration/RegistrationCart/PaymentBoard.js

```diff
@@ -337,8 +423,9 @@ export default function PaymentBoard() {
 				<Button
 					variant="contained"
 					color="primary"
-					disabled={priceQuote.total <= 0 || priceQuote.total === "-"}
+					disabled={priceQuote.total === "-" || priceQuote < 0}
```

Comment:
Why is priceQuote able to be `"-"` as well as a number?

Reply:
visually, the default is a "-", and whenever i'm getting the price quote from the backend it becomes a number